[Catalog] Unable to debug Swift code

Variables in Swift objects are not inspectable in MDCCatalog and MDCDragons.

Enter a breakpoint in an example view controller written in Swift.
For example viewDidLoad in CardExampleViewController,
CardExampleViewController.swift : 34

Run the catalog and navigate to Cards -> Start Demo
The breakpoint with fire and execution will stop.
None of the variable are inspectable.

Explicitly 'po bundle' from the CLI:
warning: Swift error in module MaterialComponentsExamples.
Debug info from this module will be unavailable in the debugger.

error: in auto-import:
failed to get module 'MaterialComponentsExamples' from AST context:
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "Headers/MaterialComponentsExamples-umbrella.h"
        ^

/Users/iangordon/Library/Developer/Xcode/DerivedData/MDCCatalog-ebygryhdxphaocekwunxdxeoorsq/Build/Products/Debug-iphonesimulator/MaterialComponentsExamples/MaterialComponentsExamples.framework/Headers/MaterialComponentsExamples-umbrella.h:13:9: note: in file included from /Users/iangordon/Library/Developer/Xcode/DerivedData/MDCCatalog-ebygryhdxphaocekwunxdxeoorsq/Build/Products/Debug-iphonesimulator/MaterialComponentsExamples/MaterialComponentsExamples.framework/Headers/MaterialComponentsExamples-umbrella.h:13:
#import "CardsCollectionExample.h"
        ^

error: /Users/iangordon/Library/Developer/Xcode/DerivedData/MDCCatalog-ebygryhdxphaocekwunxdxeoorsq/Build/Products/Debug-iphonesimulator/MaterialComponentsExamples/MaterialComponentsExamples.framework/Headers/CardsCollectionExample.h:18:9: error: 'MaterialCards.h' file not found
#import "MaterialCards.h"
        ^

error: could not build Objective-C module 'MaterialComponentsExamples'

--- a/components/AnimationTiming/examples/AnimationTimingExample.swift
+++ b/components/AnimationTiming/examples/AnimationTimingExample.swift
@@ -29,7 +29,7 @@ struct Constants {
    }
 }
 
-class AnimationTimingExample: UIViewController {
+class AnimationTimingExampleSwift: UIViewController {
 
    fileprivate let scrollView: UIScrollView = UIScrollView()
    fileprivate let linearView: UIView = UIView()
@@ -97,7 +97,7 @@ class AnimationTimingExample: UIViewController {
    }
 }
 
-extension AnimationTimingExample {
+extension AnimationTimingExampleSwift {
    fileprivate func setupExampleViews() {
 
       let curveLabel: (String) -> UILabel = { labelTitle in

